src/DataFixtures/UserFixtures.php

```diff
@@ -22,13 +25,25 @@ class UserFixtures extends Fixture
      */
     private $encoder;
 
-    public function __construct(UserPasswordEncoderInterface $encoder)
+    /**
+     * @param UserPasswordEncoderInterface $encoder
+     * @param UserRepository $userRepository
+     * @throws ORMException
+     * @throws OptimisticLockException
+     */
+    public function __construct(UserPasswordEncoderInterface $encoder, UserRepository $userRepository)
     {
         $this->encoder = $encoder;
+        $users = $userRepository->findAll();
+        foreach ($users as $user) {
+            $userRepository->unassignProjectManager($user);
```

Comment:
Why are you doing this in fixtures? Everything from database is removed at each load of fixtures.